docs/configuration/network_filters/tcp_proxy_filter_route.rst

```diff
@@ -0,0 +1,74 @@
+.. _config_network_filters_tcp_proxy_route:
+
+Route
+=====
+
+A TCP proxy route consists of a set of optional L4 criteria and the name of a :ref:`cluster manager <arch_overview_cluster_manager>` to use for a connection if it mateches all the specified criteria. Routes are tried in the order specified until a match is found. If no match is found, the connection is closed. A route with no criteria is valid and always produces a match.
```

Comment:
also, nit, please try to wrap docs to approximately 100 cols, same in other file.